.github/workflows/cpp-packaging.yml

```diff
@@ -433,6 +433,8 @@ jobs:
         run: |
           cd out-sdk
           find .. -type f -print > src_file_list.txt
+          # Remove intermediate build files (.o and .obj) files to save space.
+          find . -type f -name '*.o' -or -name '*.obj' -print0 | xargs -0 rm -f --
```

Comment:
Drive-by comment: Would it make sense to run `df` before and after this command so you could quantify how much disk space was actually saved?

Reply:
Yeah that probably would have been a good idea. :)